### DISCUSSION TITLE
La Rosa de los sueños

### DISCUSSION THREAD

**Discussion by DeepGlowing** [gamerevar/proyecto-comunitario-1-gd#57] [Archivados]
Objetivo del personaje A "Cassian": Intentar retener las raíces de la Rosa negra para “reprimir” su recuerdo, conteniéndola en la montaña de los deseos.
Objetivo del personaje B "Rose": Buscar extender las raíces por todo el Mundo de los Sueños para hacer que la flor florezca y hacer que Cassian acepte su muerte.
Cassian no sabe que La Rosa negra se alimenta del Lago de las Memorias, este se ha empezado a secar debido a ella y todo el Mundo de los Sueños empieza a decaer y desvanecerse (junto con la voluntad y personalidad de Cassian)

**Juego de Cassian Clímax:**
 Cassian vence a Rose al final de la historia y logra contener la Rosa Negra en el interior de la montaña (deseando que nunca hubiera muerto) pero con un indicio de que la Rosa Negra es la causante de la sequia del lago y esta seguirá consumiéndolo desde el interior de la montaña, ahora inaccesible para detener y encerrada en la oscuridad donde nunca florecerá.

**Juego Rose Clímax:** 
Rose vence a Cassian, pero sin intenciones de acabar con él hace que las raíces lo amarren, mientras deja expandir libremente todas las raíces por el mundo hasta llegar al Lago de las Memorias, Cassian es consumido por las raíces y el agua del lago se envenena, con esto la Rosa Negra se marchita y Rose desaparece, sin ella Cassian ya no podrá salir de debajo de las raíces. (Rose se ha ido y Cassian acepta su muerte, pero con el lago vacío ya no tiene voluntad para vivir)

**Final verdadero conjunto:**
Durante el combate Cassian reconoce a Rose, pero parte de la arena de combate desaparece debido a al efecto del lago. Rose cae, pero Cassian logra atraparla, ella le pide que **la deje ir**, que una vez la Rosa Negra florezca el mundo de los sueños dejara de desvanecerse y para ello necesita del lago. Llegan al lago y Rose guía la raíces hacia él. La enorme flor encima de La Montaña de los Deseos florece dejando atrás su color negro, pasando a ser un rojo cálido, los pétalos se esparcen por todo el mundo y reconstruyen las áreas que habían sido consumidas. Al ver esto, Cassian voltea a ver a Rose, pero ella ya no está, frente al lago solo esta su túnica, Cassian la toma y debajo de ella hay una rosa, él la toma y la vuelve a colocar en una tumba, pero ahora ya no esta en el mundo de los sueños, es el mundo real. Fin. (Dios que cursi)

Para acceder al final verdadero se debería de completar los 2 juegos, al final de cada uno se obtendría un ítem con un código, el ítem podría ser la mitad de una fotografía de Cassian y la Rose real, el código seria la fecha de su muerte.

**Aclaración de la historia**
La historia va sobre Cassian, quien parece haber perdido a quien amaba. El juego es una representación de la lucha interna de Cassian por aceptar la muerte de Rose.
Cambiemos el nombre de "Rosa Negra" a "Flor Negra" para no confundirlo con el personaje Rose. Rose es la personificación del ser querido de Cassian, ella murió. La Flor Negra es la representación de Cassian de aferrarse a ese recuerdo, el lago se puede representar como su cordura. Cassian cree que puede vivir manteniendo ese recuerdo "encerrándolo en la montaña" pero el no sabe que a la larga la Flor Negra consumirá todo el lago "su cordura".
Rose quiere que Cassian se olvide de ella y siga adelante con su vida, en el final del juego de Rose, ella lo consigue, pero sin el consentimiento real de Cassian, por eso el lago se envenena, la Flor muere porque el recuerdo ya no importa, Cassian acepta que Rose ya no esta con él, pero en vez de seguir su vida, ahora ya no tiene una razón para seguir adelante.
El desenlace del verdadero final seria la aceptación por parte de Cassian y seguir con su vida, pero aun así veríamos como aun sigue yendo a su tumba pero ya no como un mal recuerdo sino uno mas agradecido de lo que vivió con ella.
(Aclaro que todo ocurre en un mundo ficticio imaginario, solo el final donde esta la tumba y la rosa seria el mundo real)

La idea de un “mundo de los sueños” puede ser utiel para el desarrollo de este proyecto, ya que es un lugar abstracto y sin límites, esto le permitiría a cada equipo representarlo como quiera, quizás, con paletas de colores en común para que no desentonen tanto.
Esto es solo una simple idea sin desarrollar, quizás es muy ambiciosa, si en el grupo hay alguien que le guste escribir o narrar historias podría contactarme para profundizarla más, yo aún estoy estudiando estos temas.
Algo mas sencillo seria que Rose y Cassian sean ajenos a la historia que les acabo de exponer. En este nuevo caso serian 2 guardianes del mundo de los sueños y juntos se enfrentarían al señor de las pesadillas de turno. No hay historia compleja, ni giros de guion, solo un final en donde el segundo personaje debería de aparecer como un NPC en el combate final contra el Nightmare Lord (no sabía que el WoW tenía un personaje así hasta ahora) o hacer que el segundo personaje valla apareciendo en ciertos niveles para poder superar un obstáculo o puzzle, ejemplo: Tomando el mismo concepto de Rose como Dríade/Maga de rosas, que permita darle a Cassian (personaje genérico con espada) un empujón o desplazamiento especial con sus raíces, convirtiéndola así en un “objeto de entorno” en el cual el jugador de Cassian deba interactuar para superar el obstáculo del nivel.

**Comment by jh62** [comment]
La setting de la historia está bueno, se puede desarrollar bastante!

Me gusta además de que la pista para jugar los finales es a través de un item, esto se puede integrar al juego son necesitar de una pantalla colgada que diga "ingresá la clave" o "te falta jugar en otro final para continuar", etc. Tranquilamente se puede dejar como un puzzle más, sin pistas, y si no pones la fecha correcta directamente el juego sigue hacía el final normal.

**Comment by azagaya** [comment]
@DeepGlowing Me parece interesante pero no entiendo bien la historia. Cassian sería un personaje, que se le murio un ser querido. Este ser querido es la rosa negra? Por que contener la rosa negra en la montaña reprime su recuerdo? Y Rose quiere que la rosa negra extienda sus raices por el mundo?

**Reply by DeepGlowing** [reply]
Cambiemos el nombre de "Rosa Negra" a "Flor Negra" para no confundirlo con el personaje Rose. Rose es la personificación del ser querido de Cassian, ella murió. La Flor Negra es la representación de Cassian de aferrarse a ese recuerdo, el lago se puede representar como su cordura. Cassian cree que puede vivir manteniendo ese recuerdo "encerrándolo en la montaña" pero el no sabe que a la larga la Flor Negra consumirá todo el lago "su cordura".
Rose quiere que Cassian se olvide de ella y siga adelante con su vida, en el final del juego de Rose, ella lo consigue, pero sin el consentimiento real de Cassian, por eso el lago se envenena, la Flor muere porque el recuerdo ya no importa, Cassian acepta que Rose ya no esta con él, pero en vez de seguir su vida, ahora ya no tiene una razón para seguir adelante.
El desenlace del verdadero final seria la aceptacion por parte de Cassian y seguir con su vida, pero aun así veríamos como aun sigue yendo a su tumba pero ya no como un mal recuerdo sino uno mas agradecido de lo que vivió con ella.
(Aclaro que todo ocurre en un mundo ficticio imaginario, solo el final donde esta la tumba y la rosa seria el mundo real)

**Comment by LeoMast** [comment]
Hola, la historia me gusta, sobre todo los giros que pega. Tengo muchos problemas con lo abstracto y/o onírico sobre todo si requiere de un análisis o interpretación.  Personalmente me gusta la idea de robar "algo" del mundo de los sueños  para que un rey pueda resolver  un problema o ganar una guerra. De esta forma tenemos a 2 guerreros de 2 reinos distintos en busca de " La Vara de la Ilusión".
No tiene porque se una historia plana o sin giros. Se podría usar lo del FINAL VERDADERO ingresando un código que  se obtiene al juntar las dos mitades obtenidas una en cada juego.
En el Final un Guardian advierte que "La Vara de la Ilusión" no debe ser utilizada, ya que el ser humano tiene una relación muy compleja con la Ilusión que depende un equilibro inestable por lo que no se debe manípular.
DE todas formas se la llevan perooo.......
Se me ocurre que el Guerrero del Reino A obtiene "La Vara de la Ilusión" Primero pero su Rey la utiliza mal y deja a todo el mundo en un estado de ilusión permanente donde todos siente que algo les falta y deben obtenerlo del mundo que los rodea, sufriendo por una insatisfacción constante debido a que tiene una idea falsa de la felicidad o del  logro personal.
El Guerrero del reino B obtiene "La Vara de la Ilusión" pero es utilizada por el consejero de Guerra del Rey para derrocarlo y ser el Emperador de todos los reinos.  Lo logra sumiendo a las personas en distintas ilusiones   según su conveniencia. Su codicia  crece sin limites al usar la vara y esto lleva al mundo a un estado Caótico.

El final Verdadero puede ser uno en el que los guerreros deciden no llevarse la vara que si algun elemento menos poderoso que permita a sus reinos salir de la miseria. 
O tal vez algo mas drástico  en una pelea por obtener la vara, ambos guerreros, sufren ilusiones emanadas de la propia  "La Vara de la Ilusión", como una trampa para evitar intrusos. que los llevan a seguir peleando entre si hasta que ambos mueren,

**Reply by DeepGlowing** [reply]
Me gusta la idea de la vara, en especial la idea de que "el ser humano tiene una relaciona muy compleja con la ilusión", tiene algo de verdad. Pero no termino de ver el final, si ambos guerreros mueren y ninguno tiene la Vara, para que contar la historia si al final no ah cambiado nada ? Amenos que la idea de no adquirir la Vara sea una representación de algo. Prueba de poner el mismo comentario pero como post, capas que alguien logre explayar más la trama o le des alguna idea a alguien.
